### PR TITLE
Update Tournament Article Template

### LINKED ISSUE #1580: Tournament template should be updated
It should be used for future tournaments to avoid such inconsistency. It also should be linked to the ASC or whatever place, so people will know where it is.

It'll move to a folder for multiple languages, so `Tournaments/Template/en.md` maybe.

### PR DESCRIPTION
**Pull request by fajar13k**
Update old Tournament Article Template

- Closes #1580 
- Create Subfolder `Template` in `Tournaments` directory.
- Delete old TEMPLATE.md
- Create template (en, id).
- Reference Tournament Article Template in `See also` section on Tournaments index page.

---

### CODE REVIEW

File: wiki/Tournaments/Template/en.md [hunk 1]
```diff
@@ -5,54 +5,60 @@ tags:
...
 {TOPIC_NUM} = forum thread number
 {STATS_LINK} = link to overall statistics (should be Google Drive link)
 you will still to add other items such as tables
 -->
 
-# {MODE} World Cup 20{XX}
+# {TOURNAMENT_NAME} World Cup 20{XX}
 
-![{MODE} World Cup 20{XX} logo](logo.png)
+![{TOURNAMENT_NAME} World Cup 20{XX} Logo](logo.jpg)
```
Comment (Joehuu) [inline wiki/Tournaments/Template/en.md]: `{TOURNAMENT_NAME} World Cup 20{XX}` -> `{ABBR} 20{XX}`

File: wiki/Tournaments/Template/en.md [hunk 2]
```diff
@@ -5,54 +5,60 @@ tags:
...
+{ABBR} = abbreviation of the title i.e. MWC
 {XX} = year, last two digits
 {NTH} = installment ordinal number
 {TOPIC_NUM} = forum thread number
 {STATS_LINK} = link to overall statistics (should be Google Drive link)
 you will still to add other items such as tables
 -->
 
-# {MODE} World Cup 20{XX}
+# {TOURNAMENT_NAME} World Cup 20{XX}
```
Comment (Joehuu) [inline wiki/Tournaments/Template/en.md]: `{TOURNAMENT_NAME} World Cup 20{XX}` -> `{TOURNAMENT_NAME}`

Are you formatting this for community tournaments too? If so, do this change and fix other instances.

File: wiki/Tournaments/Template/en.md (hunk 1 above)
Comment (TPGPL) [reply wiki/Tournaments/Template/en.md]: or only `{ABBR}` if for community tournaments too

**Comment by Joehuu** [comment]
Enhance it how?

File: wiki/Tournaments/Template/en.md [hunk 3]
```diff
@@ -1,58 +1,63 @@
...
 
 ## Tournament Schedule
 
 | Event | Timestamp |
-|---:|---|
-| Registration Phase |  |
-| Drawings |  |
+| --: | :-- |
+| Registration Phase | 2018-01-01/2018-01-28 |
+| Live Drawings | 2018-01-29 (12 UTC) |
```
Comment (Joehuu) [inline wiki/Tournaments/Template/en.md]: Idk if this is the default format. I've seen differences in formatting on OWC and MWC.

Comment (Joehuu) [reply wiki/Tournaments/Template/en.md]: I guess we can go with the recent one.

File: wiki/Tournaments/Template/en.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+| Quarterfinals |  |
+| Semifinals |  |
+| Finals - Week 1 |  |
+| Finals - Week 2 |  |
+
+<!-- Use YYYY-MM-DD Format -->
+
+## Prizes
+
+In every tournament, people conquering a place on the podium are eligible to receive unique prizes created specifically for {TOURNAMENT_NAME} winners. As these items may change with every installment of the tournament, it is only possible to give an estimate prize list for each {TOURNAMENT_NAME}.
```
Comment (Joehuu) [inline wiki/Tournaments/Template/en.md]: `As these items may change with every installment of the tournament, it is only possible to give an estimate prize list for each {TOURNAMENT_NAME}.` ->
`It is possible that these items change with every installment of the {TOURNAMENT_NAME}.`

File: wiki/Tournaments/Template/en.md [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+- [Livestream](https://www.twitch.tv/osulive)
+- **[Statistics sheet]({STATS_LINK})**
+
+-----------------------------------
+
+![{ABBR} Podium](img/podium.png)
+
+![{ABBR} Bracket](img/bracket.png)
+
+---------------------------------------
```
[the remark this answers is not in the document]
Comment (fajar13k) [reply wiki/Tournaments/Template/en.md]: I forgot this spot is usually used for schedules 😆

File: wiki/Tournaments/Template/en.md [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+---------------------------
+
+## Mappools
+
+### Finals
+
+**[Download the mappack here! (SIZE)](LINK)**
+
+- ModType
+ 	- Beatmap
```
Comment (Joehuu) [inline wiki/Tournaments/Template/en.md]: I think the indent is only two spaces.

File: wiki/Tournaments/Template/en.md [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+### Tournament Rules
+### Tournament Registration
+### Stage Instructions
+### Match Instructions
+### Mappool Instructions
+### Scheduling Instructions 
+
+<!-- 
+Add flag references here
+Use https://megaapplepi.github.io/flag-wiki-osu/ for usage of flag references
```
Comment (Joehuu) [inline wiki/Tournaments/Template/en.md]: End the two sentences shown with periods.

File: wiki/Tournaments/Template/en.md [hunk 8]
```diff
@@ -0,0 +1,146 @@
...
+{TOURNAMENT_NAME} = tournament name
+{MODE} = game mode
+{ABBR} = abbreviation of the tournament name followed by its iteration/year occurence i.e. MWC 4K 2018
+{ORGANISER} = organiser of the tournament
+{NTH} = installment ordinal number
+{TOPIC_NUM} = forum thread number
+{STATS_LINK} = link to overall statistics sheet (it should be a Google Drive link)
+You will still need to fill other items such as tables, etc.
+Use .png as a default format for images, since .jpg can't handle transparency.
+Alongside this comment, remember you will need to delete all comments across the article.
```
Comment (TPGPL) [inline wiki/Tournaments/Template/en.md]: remember  to delete all comments across the article after you finish

File: wiki/Tournaments/Template/en.md [hunk 9]
```diff
@@ -0,0 +1,146 @@
...
+![{ABBR} Logo](img/logo.png)
+
+The **{TOURNAMENT_NAME}** (***{ABBR}***) is a country-based {MODE} tournament hosted by the {ORGANISER}. It is the {NTH} installment of the {TOURNAMENT_NAME}.
+
+## Tournament Schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration Phase | 2018-01-01/2018-01-28 |
+| Live Drawings | 2018-01-29 12:00:00 UTC |
```
Comment (TPGPL) [inline wiki/Tournaments/Template/en.md]: sure we are going with `12:00:00 UTC`?

Comment (fajar13k) [reply wiki/Tournaments/Template/en.md]: Or maybe i should go with placeholder?
Since it's up to the organiser

Comment (Joehuu) [reply wiki/Tournaments/Template/en.md]: I think we were talking about the format here if it should have the minutes and no parenthesis.

As with the random numbers, it should be a placeholder. Like `YYYY-MM-DD HR:MIN:SEC`?

Also idk why seconds were included.

Comment (fajar13k) [reply wiki/Tournaments/Template/en.md]: juan used the `YYYY-MM-DD HR:MIN:SEC` for recent 4K WC 2018.
But i don't know, which is more suitable in this case. Should i remove `SEC`?

File: wiki/Tournaments/Template/en.md [hunk 10]
```diff
@@ -0,0 +1,149 @@
...
+
+- [Discussion thread](/community/forums/topics/{TOPIC_NUM})
+- [Livestream](https://www.twitch.tv/osulive)
+- **[Statistics sheet]({STATS_LINK})**
+
+## Participants
+
+|  | Country | Members |
+| :-: | :-: | :-- |
+| ![][flag_AU] | **Australia** | example; delete this line |
```
Comment (Joehuu) [inline wiki/Tournaments/Template/en.md]: Probably use the [unknown flag (not in `/shared/flag` yet)](https://github.com/ppy/osu-web/blob/master/public/images/flags/__.png), and change the countries/users to `Country1`, `User1`?

**Comment by fajar13k** [comment]
Anything to point out again ?

File: wiki/Tournaments/Template/en.md [hunk 11]
```diff
@@ -0,0 +1,149 @@
...
+You will still need to fill other items such as tables, etc.
+Use .png as a default format for images, since .jpg can't handle transparency.
+Alongside this comment, remember to delete all comments across the article after you finish.
+-->
+
+# {TOURNAMENT_NAME}
+
+![{ABBR} Logo](img/logo.png)
+
+The **{TOURNAMENT_NAME}** (***{ABBR}***) is a country-based {MODE} tournament hosted by the {ORGANISER}. It is the {NTH} installment of the {TOURNAMENT_NAME}.
```
Comment (Joehuu) [inline wiki/Tournaments/Template/en.md]: `{NTH} installment of the {TOURNAMENT_NAME}` -> `{NTH} installment of the {TOURNAMENT_SERIES}`

`SERIES` is the one without the iteration suffix.

File: wiki/Tournaments/Template/en.md [hunk 12]
```diff
@@ -0,0 +1,150 @@
...
+| Finals - Week 1 |  |
+| Finals - Week 2 |  |
+
+<!-- Use YYYY-MM-DD Format for dates i.e. 2018-01-01
+Use (HH:MM) format for {DRAW_TIME} i.e. (12:00 UTC)
+ -->
+
+## Prizes
+
+In every tournament, people conquering a place on the podium are eligible to receive unique prizes created specifically for {TOURNAMENT_NAME} winners. It is possible that these items change with every installment of the {TOURNAMENT_NAME}.
```
Comment (Joehuu) [inline wiki/Tournaments/Template/en.md]: `every installment of the {TOURNAMENT_NAME}` -> `every installment of the {TOURNAMENT_SERIES}`

File: wiki/Tournaments/en.md [hunk 13]
```diff
@@ -146,6 +146,7 @@ Unofficial tournaments/competitions hosted by the communities.
 See also
 --------
 
+-   [Tournament articles formatting](/wiki/Tournaments/Template)
```
Comment (TPGPL) [inline wiki/Tournaments/en.md]: Tournament article template

Comment (TPGPL) [reply wiki/Tournaments/en.md]: or articles

**Pull request merged**